ocp_resources/rhmi.py

```diff
@@ -8,3 +9,43 @@ class RHMI(NamespacedResource):
     """
 
     api_group = NamespacedResource.ApiGroup.INTEGREATLY_ORG
+
+    class Status:
+        INSTALLATION = "installation"
+        COMPLETE = "complete"
```

Comment:
`ocp_resources/installplan.py` also have `COMPLETE`
Please move it to resource.py and use it from there (for both)